log/logrus_writer_test.go

```diff
@@ -298,11 +297,11 @@ func TestLogrus(t *testing.T) {
 			assert: func() {
 				expectedParts := []string{
 					`test.info: Goravel`,
-					`response: {`,
-					`"status":200`,
-					`"header":{"Content-Type":["text/plain; charset=utf-8"]}`,
-					`"body":{}`,
```

Comment:
@kkumar-gcc The `body` is missing when usnig `json.Marshal`, and `map[any]any` can't be marshaled, golang throws an error.

Reply:
@hwbrzzl I tested it, and the body is not missing, but `map[any]any` cannot be parsed.

![image](https://github.com/user-attachments/assets/1d627ce6-184a-4c09-a873-a9398ddb7b2a)

Reply:
@hwbrzzl I believe the reason the body is missing is that we should use `res.Origin().Body().String()` because `res.Origin().Body()` is a buffer.
However, `map[any]any` cannot be parsed.

Reply:
Yes, so we don't need to use `json.Marshal`, printing them should be fine.

Reply:
Well, it will include `map` and other type information, which I’m not sure is a good idea to include in the logs.

Reply:
Yes, it's fine, I prefer to know the specific type.